docs/en/latest/discovery/consul_kv.md

```diff
@@ -23,7 +23,7 @@ title: consul_kv
 
 ## Summary
 
-For users who used [nginx-upsync-module](https://github.com/weibocom/nginx-upsync-module) and consul key value for service discovery way, as we Weibo Mobile Team, maybe need it.
+For users that are using [nginx-upsync-module](https://github.com/weibocom/nginx-upsync-module) and Consul KV as a service discovery, like the Weibo Mobile Team, this may be required.
```

Comment:
Why change `maybe need it`? We just suggest people use but not enforce them to use.

Reply:
@spacewander Sorry, in my opinion, "may be required" doesn't mean "enforce". If I change it to "this may be needed", do you think it will be better? Or do I change back?

Reply:
ok